vantage6/cli/server.py

```diff
@@ -104,17 +110,18 @@ def cli_server():
 @cli_server.command(name='start')
 @click.option('--ip', default=None, help='ip address to listen on')
 @click.option('-p', '--port', default=None, type=int, help='port to listen on')
-@click.option('--debug', is_flag=True,
-              help='run server in debug mode (auto-restart)')
 @click.option('-i', '--image', default=None, help="Server Docker image to use")
+@click.option('--rabbitmq-image', default=None,
+              help="RabbitMQ docker image to use")
```

Comment:
No default option? Do we now always have to specify an image name?

Reply:
There is a default image: https://github.com/IKNL/vantage6/blob/feature/rabbitmq-v2/vantage6/cli/rabbitmq/queue_manager.py#L20. We could choose to specify that image there. I didn't do it for consistency: we also didn't do it for `--image`. However, we probably will not change the rabbitmq image that often...